packages/approval-controller/src/ApprovalController.test.ts

```diff
@@ -462,7 +462,8 @@ describe('approval controller', () => {
 
   describe('get', () => {
     it('gets entry', () => {
-      approvalController.add({
+      // eslint-disable-next-line no-void
+      void approvalController.add({
```

Comment:
Can we `await` this instead, since it returns a promise?

Reply:
Thanks for the feedback Mark, I've sorted this in https://github.com/MetaMask/core/pull/1728/commits/b38a4714ff3d65fd40ce875d24a1a286dbb8c43a